receiver/jaegerreceiver/factory.go

```diff
@@ -128,11 +128,23 @@ func createDefaultConfig() configmodels.Receiver {
 			ThriftHTTP: &confighttp.HTTPServerSettings{
 				Endpoint: defaultHTTPBindEndpoint,
 			},
-			ThriftBinary: &confignet.TCPAddr{
+			ThriftBinary: &ProtocolUDP{
 				Endpoint: defaultThriftBinaryBindEndpoint,
+				ServerConfigUDP: ServerConfigUDP{
+					QueueSize:        defaultAgentQueueSize,
```

Comment:
Where are these constants defined? As they are primarily set in this file now, it would be better to move them here.

Reply:
They're defined in receiver file. Sure, I'll move them to factory